yogstation/code/modules/mob/living/carbon/human/species_types/preternis/power_suck.dm

```diff
@@ -112,6 +112,8 @@
 
 //IPC lol, lmao
```

Comment:
your PR says "preternis" but the comments says "IPC"

Reply:
it's me laughing at IPCs after draining electricity from them as a preternis